examples/with-thirdpartypasswordless/src/App.js

```diff
@@ -1,6 +1,6 @@
 import { useState } from "react";
 import "./App.css";
-import SuperTokens, { getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
+import SuperTokens, { SuperTokensWrapper, getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
```

Comment:
- Need to update package.json in all apps too to use the latest version of SDK.
- Have you tested all the apps in which you have made changes to?

Reply:
- I thought that should be done when it was getting released. Should I do it now?
- I just ran through them with a few exceptions (supabase, AWS, hasura, netlify, with-next-iframe). There are a few cases where I ran into issues, but they are unrelated to this PR. I'll add an issue to test, check and fix the example apps.

Reply:
#516